### PR TITLE
Appveyor

### PR DESCRIPTION
**Pull request by chrishelgert**
appveyor will run the ci on windows, so we can make sure, that everything is working on unix and windows.

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
```
Comment (danielbayerlein) [inline .appveyor.yml]: TODO: Remove empty line

File: .appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  matrix:
```
Comment (danielbayerlein) [inline .appveyor.yml]: TODO: Incorrect line indentation

Comment (danielbayerlein) [inline .appveyor.yml]: `Error parsing appveyor.yml: (Line: 6, Col: 1, Idx: 64) - (Line: 6, Col: 1, Idx: 64): Did not find expected <document start>.`

File: .appveyor.yml (hunk 1 above)
Comment (chrishelgert) [inline .appveyor.yml]: sry looks like i lost one line :/

**Pull request merged**